blog/_posts/2022-12-23-wildcart-domain-with-caddy-for-tunnel-hosting.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: Use a wildcard domain with Caddy for tunnel hosting.
```

Comment:
> Use a wildcard domain with Caddy for tunnel hosting.

Very technical

What is the solution-orientated title?

Problem: I want to host different sites on one server with quick configuration.

How to expose/host [multiple] sites [for development] on a server with inlets [tunnels] and [Caddy]